extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.webjar.locator.deployment;
+
+import java.util.Map;
+
+import org.webjars.WebJarAssetLocator;
+
+import io.quarkus.deployment.Capabilities;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.vertx.http.deployment.RouteBuildItem;
+import io.quarkus.vertx.http.runtime.HttpBuildTimeConfig;
+import io.quarkus.webjar.locator.runtime.WebJarLocatorHandler;
+
+public class WebJarLocatorStandaloneBuildStep {
+
+    @BuildStep()
+    public void staticInit(Capabilities capabilities,
```

Comment:
The name doesn't seem really fitting :)

Reply:
The name was copied from resteasy extension. How about `findWebjarsAndRootPath`?

Reply:
Sure :)